src/global_distrib/mod_global_distrib_server_mgr.erl

```diff
@@ -297,7 +297,7 @@ get_endpoints(Server) ->
         undefined -> mod_global_distrib_mapping:endpoints(Server);
         Endpoints -> {ok, Endpoints}
     end,
-    {ok, mod_global_distrib_utils:resolve_endpoints(EndpointsToResolve)}.
+    {ok, EndpointsToResolve}.
```

Comment:
@fenek, @arcusfelis, All tests pass after I deleted it, but could you, please, take a look and tell whether it is ok?

Reply:
Ok, so we probably don't have a test for `global_distrib_addr` and that is a case where this would fail after deletion of this function call